pkg/cluster/px/px.go

```diff
@@ -39,12 +54,15 @@ const (
 	pxdRestPort        = 9001
 	pxServiceName      = "portworx-service"
 	pxClusterRoleName  = "node-get-put-list-role"
+	pxVersionLabel     = "PX Version"
```

Comment:
could this string be more kubernetes-like? Like px-version, or something like that? Just a suggestion

Reply:
This label is populated by Portworx (porx) on the node labels. We simply read it here.